rollup.config.js

```diff
@@ -8,10 +8,16 @@ export default [
     output: [
       {
         name: 'Diff',
-        file: pkg.browser,
-        format: 'umd'
+        format: 'umd',
+        file: pkg.browser
       },
-      { file: pkg.module, format: 'es' }
+      {
+        format: 'esm',
+        file: pkg.module
+      }, {
+        format: 'esm',
+        file: pkg['exports']['.'].import
+      }
```

Comment:
@lukeed What is the advantage of having two separate ESM builds?

Reply:
Two things:

1) I'm primarily preserving `diff`'s existing output files, so that anyone who was import/requiring into the module (eg, `require('diff/lib/index.es6.js')`) does find a surprise.

2) I did not add `"type": "module"` (nor do I want to) since that'd be a breaking change. Without that, need `.mjs` extension for ESM to be activated.

Reply:
Thanks, that makes sense!